python/mxnet/ndarray/numpy/_op.py

```diff
@@ -27,7 +27,8 @@
 from ..ndarray import NDArray
 
 __all__ = ['zeros', 'ones', 'add', 'subtract', 'multiply', 'divide', 'mod', 'power', 'tensordot',
-           'linspace', 'expand_dims', 'tile', 'arange', 'split', 'concatenate', 'stack']
+           'linspace', 'expand_dims', 'tile', 'arange', 'split', 'concatenate', 'stack', 'full',
+           'slice_assign_scalar', 'slice_assign']
```

Comment:
We should only expose numpy ops, while these two are not.

Reply:
Sure. Thx!